code/modules/mob/living/living_verbs.dm

```diff
@@ -263,9 +253,7 @@
 				for(var/mob/O in viewers(CM))
 					O.show_message( "<span class='danger'>[usr] attempts to remove [LC]!</span>", 1)
 				spawn(0)
-					if(do_after(CM, breakouttime, FALSE, 5, BUSY_ICON_HOSTILE))
```

Comment:
Is the removal of the icon intended here?

Reply:
idem

Reply:
You shouldn't change functionality here